### PR TITLE
Fix: Allow register of private repos (fixes #200)

### LINKED ISSUE #200: Will not register private: true bower.json
### Subject of the issue
By adding automated releases to each plugin, and causing the bower.json to be a copy of the package.json and in including `"private": true` in the package.json to prevent the automation scripts publishing on npm, the bower library will now not register the plugins, due to these lines in the bower library https://github.com/bower/bower/blob/a0d44443245cbe52f3f0bd90c3f41274bc040c7a/lib/commands/register.js#L44-L49

As we register plugins by hand, it should be possible to register plugins in our bower repos even if they are marked as private.

### Example package.json:
```js
  "private": true,
  "scripts": {
    "postversion": "cp package.json bower.json"
  },
```

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #200 

### Fix
* Allow register of private repos

### CODE REVIEW

**Review by swashbuck — APPROVED** [review]
👍

File: lib/integration/PluginManagement/register.js [hunk 1]
```diff
@@ -120,11 +121,19 @@ async function exists (BOWER_REGISTRY_CONFIG, plugin) {
...
+  const response = await fetch(`${BOWER_REGISTRY_CONFIG.register}packages`, {
+    headers: {
+      'User-Agent': 'adapt-cli',
+      'Content-Type': 'application/json'
+    },
+    followRedirect: false,
+    method: 'POST',
+    body: JSON.stringify(data)
   })
+  if (response.status !== 201) return false;
```
Comment (cahirodoherty-learningpool) [inline lib/integration/PluginManagement/register.js:137]: Minor: 
```suggestion
  return response?.status === 201;
```

**Review by joe-allen-89 — APPROVED** [review]
👍

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**